### ISSUE TITLE
german babelshorthands inside latex3 class produces error

### ISSUE THREAD

**Issue by stev47** [reutenauer/polyglossia#67]
I'm compiling the following code with `xelatex`

``` tex
\begin{filecontents}{myclass.cls}

    \RequirePackage {expl3, l3keys2e, xparse}
    \ProvidesExplPackage {myclass} {2013/01/01} {0.1} { }

    \RequirePackage {fontspec}

    \RequirePackage {polyglossia}
    \setmainlanguage [babelshorthands=true] {german}

    \LoadClass{scrbook}

\end{filecontents}

\documentclass{myclass}
\begin{document}
    test
\end{document}
```

getting the error

```
(/usr/share/texmf-dist/tex/latex/polyglossia/babelsh.def
! Improper alphabetic constant.
<to be read again> 
                   \string 
l.403 \initiate@active@char{~}

?
```

I'm using recent texlive packages.

**Comment by bastien-roucaries** [comment]
@reutenauer  Could be documented but not a bug

**Comment by jspitz** [comment]
not a bug.

**Comment by jspitz** [comment]
After all, it was fixable and will be fixed in polyglossia 1.50. See 96afd98411dceb7.
